### PR TITLE
Add Neoload 7.4

### PR DESCRIPTION
**Pull request by QAInsights**
Add Neoload 7.4

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1455)

### CODE REVIEW

File: manifests/Neotys/NeoLoad/7.3.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Id: Neotys.NeoLoad
+Name: NeoLoad
+AppMoniker: neoloadgit
```
Comment (samiranshah) [inline manifests/Neotys/NeoLoad/7.3.yaml:3]: Please make this field consistent in both files?

Comment (QAInsights) [reply manifests/Neotys/NeoLoad/7.3.yaml]: @samiranshah I have edited 7.3 and submitted a PR.

Comment (QAInsights) [reply manifests/Neotys/NeoLoad/7.3.yaml]: There was a typo in the AppMoniker in 7.3.

**Review by QAInsights — COMMENTED** [review]
I have fixed the typo.

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**